content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+- [Ztunnels](/blog/2023/rust-based-ztunnel/) are a brand new Istio component written in Rust that are designed to be fast, secure, and lightweight. Ztunnels are deployed per node on a cluster and enable the most basic service mesh configurations for L4 features such as mTLS, telemetry, authentication, and L4 authorizations.
+- [Waypoint proxies](/blog/2023/waypoint-proxy-made-simple/) provide L7 mesh features such as VirtualService routing, L7 telemetry, and L7 authorizations policies. Waypoints are still based on Envoy and are deployed at the namespace level per ServiceAccount.
+
+These ztunnels and waypoint proxies work in tandem to replace sidecars in the Istio service mesh. So let’s take a closer look at how the two architectures compare in the tests above.
+
+## A Closer Look
+
+{{< image width="100%"
+    link="cpu-by-pod.png"
+    caption="CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod.png"
+    caption="Memory usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="resource-usage.png"
+    caption="CPU and Memory usage for cluster"
+    >}}
+
+Let’s start by looking at CPU usage by pod. In the sidecar scenarios, the containers utilizing the most CPU resources are the [frontend](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/frontend) service sidecars, which is main server hit by the load generators and also responsible for making any subsequent calls to other underlying microservices.
+
+In the ambient scenarios, a particular ztunnel's performance depends on the distribution of services across the nodes in the cluster. In the ambient with both ztunnel and waypoint proxy scenario, there is an increase in average CPU usage per ztunnel as traffic is now directed to waypoint proxies. The waypoint proxies themselves consume more resources than their sidecar counterparts since there are only three instances processing all traffic in the namespace versus one per pod in the sidecar scenario.
+
+Next is memory usage by pod. In all Istio scenarios, memory usage stays relatively constant for each pod during the test runs. In both ambient scenarios, the L4 ztunnel consumes less memory than a sidecar. In fact, in the L4 only scenario, almost all three ztunnel instances consume less memory than a single sidecar. Waypoint proxies consume a similar amount of resources as sidecars do, but once again see a minor increase due to the same reasons outlined above for CPU. So what do these per pod usages mean for the totals across the cluster? Everything.
+
+{{< image width="100%"
+    link="total-cpu.png"
+    caption="Total CPU usage for sidecar and ambient scenarios"
+    >}}
+
+{{< image width="100%"
```

Comment:
note: you used "total mem used" in this image while you used "max mem used" earlier (first table). can you be consistent to avoid confusion?

Also would be good to explain to reader how total is calculated.  I think it is multiple 144 (for sidecars) or multiple 3 (ztunnel) etc.

you could explain it in the table, for example 144 * 39.93 = 5750Mb

Reply:
Note: sorry about commenting in the wrong table.... a little hard to co-relate. above comment is for the CPU and Memory usage for cluster table.